.clippy.toml

```diff
@@ -0,0 +1 @@
+type-complexity-threshold = 9000
```

Comment:
I don't want this lint to run and this is the easiest way to disable it without adding clippy tags to the source code (which I don't want to do).